docs/analysis/web.md

```diff
@@ -66,9 +66,32 @@ It is possible to use a POST or PUT requests, by specifying the `method` and `bo
             value: "Bearer {{ args.api-token }}"
           - key: Content-Type # if body is a json, it is recommended to set the Content-Type
             value: "application/json"
-        body: "{\"key\": \"string value\"}"
+        body: "string value"
         jsonPath: "{$.data.ok}"
 ```
   !!! tip
-      In order to send in JSON, you have to encode it yourself, and send the correct Content-Type as well.
-      Setting a `body` field for a `GET` request will result in an error.
+      In order to send in JSON, you can use jsonBody and Content-Type will be automatically set as json.
+      Setting a `body` or `jsonBody` field for a `GET` request will result in an error.
+      Set either `body` or `jsonBody` and setting both will result in an error.
```

Comment:
Is this sentence needed `Set either `body` or `jsonBody` and setting both will result in an error.`?

Reply:
@mayzhang2000 Setting both will result in an error. Should I be removing this line from the doc?